src/components/indicators/indicators.js

```diff
@@ -129,47 +119,42 @@ define(['datetime', 'itemHelper', 'emby-progressbar', 'css!./indicators.css', 'm
 
         if (item.SeriesTimerId) {
             if (status !== 'Cancelled') {
-                return '<i class="material-icons timerIndicator indicatorIcon fiber_smart_record"></i>';
+                return '<span class="material-icons timerIndicator indicatorIcon fiber_smart_record"></span>';
             }
 
-            return '<i class="material-icons timerIndicator timerIndicator-inactive indicatorIcon fiber_smart_record"></i>';
+            return '<span class="material-icons timerIndicator timerIndicator-inactive indicatorIcon fiber_smart_record"></span>';
         }
 
-        return '<i class="material-icons timerIndicator indicatorIcon fiber_manual_record"></i>';
+        return '<span class="material-icons timerIndicator indicatorIcon fiber_manual_record"></span>';
     }
 
-    function getSyncIndicator(item) {
+    export function getSyncIndicator(item) {
         if (item.SyncPercent === 100) {
-            return '<div class="syncIndicator indicator fullSyncIndicator"><i class="material-icons indicatorIcon file_download"></i></div>';
+            return '<div class="syncIndicator indicator fullSyncIndicator"><span class="material-icons indicatorIcon file_download"></span></div>';
         } else if (item.SyncPercent != null) {
-            return '<div class="syncIndicator indicator emptySyncIndicator"><i class="material-icons indicatorIcon file_download"></i></div>';
+            return '<div class="syncIndicator indicator emptySyncIndicator"><span class="material-icons indicatorIcon file_download"></span></div>';
         }
 
         return '';
     }
 
-    function getTypeIndicator(item) {
-        if (item.Type === 'Video') {
-            return '<div class="indicator videoIndicator"><i class="material-icons indicatorIcon">videocam</i></div>';
-        }
-        if (item.Type === 'Folder') {
-            return '<div class="indicator videoIndicator"><i class="material-icons indicatorIcon">folder</i></div>';
-        }
-        if (item.Type === 'PhotoAlbum') {
-            return '<div class="indicator videoIndicator"><i class="material-icons indicatorIcon photo_album"></i></div>';
-        }
-        if (item.Type === 'Photo') {
-            return '<div class="indicator videoIndicator"><i class="material-icons indicatorIcon">photo</i></div>';
+    export function getTypeIndicator(item) {
+        const iconT = {
+            'Video' : 'videocam',
+            'Folder' : 'folder',
+            'PhotoAlbum' : 'photo_album',
+            'Photo' : 'photo'
         }
 
-        return '';
+        const icon = iconT[item.iconType];
```

Comment:
I copied from your code and missed that `item` has `Type` instead of `iconType`.
```suggestion
        const icon = iconT[item.Type];
```